src/ui/index.ts

```diff
@@ -87,7 +93,7 @@ fin.desktop.main(async () => {
             const success = await plugin.dispatch('clear-app-notifications', payload);
             console.log('success', success);
         },
-        addEventListener: async (event: string, cb: (payload: Notification|ISenderInfo, sender: ISenderInfo) => string) => {
+        addEventListener: async (event: string, cb: (payload: any, sender: ISenderInfo) => string) => {
```

Comment:
Any way we can type this?

Reply:
we may be able to reference a "shared" javascript file in both the UI and client, probably best for another smaller PR.

Reply:
yeah, lemme see, i think i may have been getting lazy.

Reply:
done and I think the types are better resolved now because of this.